tests/playerStub.js

```diff
@@ -0,0 +1,12 @@
+const Player = require('../lib/player.js');
+
+module.exports = class PlayerStub extends Player {
+  constructor(symbol) {
+    super(symbol);
+    this.moves = [];
```

Comment:
Isn't it overkill to use an array where you only use `getInput` once in your tests?